helm/cluster/templates/clusterapi/controlplane/_helpers_flatcar.tpl

```diff
@@ -0,0 +1,86 @@
+{{- define "cluster.kubeadmControlPlane.kubeadmConfigSpec.ignition.containerLinuxConfig.additionalConfig.systemd.units" }}
+{{- range $.Values.internal.controlPlane.kubeadmConfig.ignition.containerLinuxConfig.additionalConfig.systemd.units }}
+- name: {{ .name }}
+  {{- if hasKey . "enabled" }}
+  enabled: {{ .enabled }}
+  {{- end }}
+  {{- if hasKey . "mask" }}
+  mask: {{ .mask }}
+  {{- end }}
+  {{- if .contents }}
+  contents: |
+    {{- .contents | trim | nindent 4 }}
+  {{- end }}
+  {{- if .dropins }}
+  dropins:
+  {{- range .dropins }}
+  - name: {{ .name }}
+    {{- if .contents }}
+    contents: |
+      {{- .contents | trim | nindent 6 }}
+    {{- end }}
+  {{- end }}
+  {{- end }}
+{{- end }}
+{{- end }}
+
+{{- define "cluster.kubeadmControlPlane.kubeadmConfigSpec.ignition.containerLinuxConfig.additionalConfig.storage.filesystems" }}
```

Comment:
Helper template for storage filesystems configuration in KubeadmControlPlane.

This got updated in a later PR to include common storage filesystems across both CP and workers, as well as CP-specific storage filesystems.